lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -954,18 +971,20 @@ ParseResult FIRParser::parseType(FIRRTLType &result, const Twine &message) {
         parseType(type, "expected probe data type"))
       return failure();
 
-    // Probe Color
     SmallVector<StringRef> layers;
-    if (getToken().getKind() == FIRToken::identifier) {
-      if (requireFeature({3, 2, 0}, "colored probes"))
-        return failure();
-      do {
-        StringRef layer;
-        loc = getToken().getLoc();
-        if (parseId(layer, "expected layer name"))
+    if (consumeIf(FIRToken::comma)) {
+      // Probe Color
+      if (getToken().getKind() == FIRToken::identifier) {
```

Comment:
Trailing comma is kinda weird here, isn't it?

Reply:
yea.

Reply:
I backed out of allowing trailing commas.

I found a bit of a quirk with the version gating on colors. But I think I got it.